app-modules/knowledge-base/src/Filament/Resources/KnowledgeBaseItemResource/Pages/CreateKnowledgeBaseItem.php

```diff
@@ -95,19 +95,15 @@ public function form(Form $form): Form
                     ->preload()
                     ->exists((new Division())->getTable(), (new Division())->getKeyName()),
                 TiptapEditor::make('solution')
-                    ->directory('kb-images')
```

Comment:
If you wouldn't mind going to the other places this custom component is being used and remove the `directory` method call as well?